ci-operator/step-registry/baremetalds/ipi/test/baremetalds-ipi-test-chain.yaml

```diff
@@ -1,6 +1,9 @@
 chain:
   as: baremetalds-ipi-test
   steps:
-  - ref: baremetalds-e2e-test
+    test:
+    - ref: baremetalds-e2e-test
+      env:
+        PACKET_PLAN: n2.xlarge.x86
```

Comment:
Not sure if you can set environment variables on chain. But if so I would expect it to be this:

```
chain:
  as: baremetalds-ipi-test
  steps:
  - ref: baremetalds-e2e-test
  env:
    PACKET_PLAN: n2.xlarge.x86
```
A chain is used in a workflow so you can't define test here. In other places there are defaults defined at the workflow level to get a huge group of tests like this.

Reply:
Here I found this in the docs: https://docs.ci.openshift.org/docs/architecture/step-registry/#stepchainworkflowtest-does-not-accept-env-field

Reply:
I'm quite sure I've tried this already with the same result. I'll try again.